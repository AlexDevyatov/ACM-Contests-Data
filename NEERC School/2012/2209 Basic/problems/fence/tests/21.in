10
-50 23
-46 -13
-43 -35
-8 -42
45 -46
48 -26
36 34
20 45
-16 49
-48 39
1 33
