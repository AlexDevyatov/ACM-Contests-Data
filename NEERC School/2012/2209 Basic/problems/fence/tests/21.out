11
-50 23
-46 -13
-43 -35
-8 -42
45 -46
48 -26
36 34
1 33
20 45
-16 49
-48 39
